test/validate/linux_validate.go

```diff
@@ -22,6 +22,11 @@ var (
 	cnsLocalCacheCmd      = []string{"curl", "localhost:10090/debug/ipaddresses", "-d", "{\"IPConfigStateFilter\":[\"Assigned\"]}"}
 )
 
+// IPv4 overlay Linux and windows nodes must have this label
+var v4OverlayNodeLabels = map[string]string{
```

Comment:
Does the node label differ if it is v4 or v6? What other variants are there?

Reply:
you can compare with dualstackOverlayNodeLabels map